articles/logout/guides/logout-applications.md

```diff
@@ -15,85 +15,7 @@ Enterprise users typically have SSO enabled for multiple applications (e.g., Sha
-      name: 'JWT Debugger',
-      logout_url: 'http://fabrikam-jwt.azurewebsites.net/'
-    },
-    'bOFty3tWgpijnwMcltysNFqHgO1ziz1I': {
-      name: 'SharePoint Intranet',
-      logout_url: 'http://sp.fabrikamcorp.com/_layouts/15/SignOut.aspx'
-    },
-    'h2NE1kxhzzFgLpNBvcCuyefjfwFVGx49': {
-      name: 'Timesheet SaaS',
-      logout_url: 'http://fabrikam-timesheets.azurewebsites.net/account/logoff'
-    }
-  };
-```
-
-Once the users are logged out of all of the applications,  they are redirected to the logout endpoint in Auth0 to clear the SSO cookie. If any of the applications fail to respond within the configured timeout (5 sec), an error appears. For this to work, all of the applications need to have a logout endpoint accessible from a GET.
-
-### Sample scenario
-
-Mary logged in to all of the applications configured in the Fabrikam account (JWT Debugger, SharePoint Intranet and Timesheet SaaS). When she hits the "Sign-out" button in one of these applications it should redirect to the "Single-Logout application" which Fabrikam is hosting somewhere (eg: https://fabrikam-logout.azurewebsites.net).
-
-The logout application will use Mary's SSO cookie to determine from which applications she needs to sign out and render iframes pointing to the logout endpoints for each of these applications.
-
-![Mary Signing Out](/media/articles/logout/signing-out-scenario1.png)
-
-Once logout is complete in all of these applications, Mary can hit the continue button which redirects to Auth0 and removes her SSO cookie there (and optionally also sign out from the IdP, like ADFS). Finally, Mary returns to the logout appliation which shows she's been logged out.
-
-![Mary Logout Complete](/media/articles/logout/signing-out-scenario2.png)
-
-## Limitations
-
-The complete single sign-out flow is based on the SSO cookie in Auth0 and rendering iframes. Take into account that:
-
-* For the IFRAMES, there's no way to detect if the IFRAME rendered correctly. You can only detect if it completed or not. For slow applications, if it takes more than 5 seconds to sign out the sample will show a red line instead of the green line.
-
-* Depending on specific browser settings there might be issues with the SSO cookie or the rendering of the IFRAME.
-In IE for example a setting called "Protected Mode" should be disabled or your applications will need a P3P header for this to work correctly.
-
-* In Safari (depending on your settings) cookies might be restricted to a specific tab (and not shared over different tabs).
+Redirecting users to the logout endpoint **does not** cover the scenario where users need to be signed out of all of the applications they used.  If you need to provide this functionality you will have to either handle this by calling redirect to Auth0 to attempt to log in at some interval in your application, or tracking this at your application level and providing a way for one application to call the other applications to force logout. 
```

Comment:
```suggestion
Redirecting users to the logout endpoint **does not** cover the scenario where users need to be signed out of all of the applications they used.  If you need to provide this functionality you will have to handle this in one of two ways:
*  Have short timeouts on your local session and redirect to Auth0 at short intervals to re-authenticate.  NOTE: this can be done by calling checkSession from the client which does this redirect in a hidden iFrame.  If you take the hidden iFrame approach you need to be aware of rate limits and third-party cookie issues.
* You can also handle this entirely at the application level by providing your applications a way to notify all other applications when a logout occurs.
```